packages/api-client/src/odsql/clauses.ts

```diff
@@ -1,11 +1,36 @@
 import { field, string } from '.';
 
+const formatFields = (fields: string[]) => {
+    if (!fields || fields.length === 0) return '';
+    return `${fields.map(field).join(', ')},`;
+};
+
+/**
+ * Perform full-text search on all fields. It matches the text fields that contain strings beginning with the searched string.
+ * @param text_fields Fields to search in
```

Comment:
`text_fields` looks optional ?

I think the description for this arg should indicate that the search will be restricted to the fields passed.